### ISSUE TITLE
Bridge and Tunnel button artwork

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2702]
Here's what I put together for the rest of Bridge and Tunnel.  The artwork and photos are used with permission, and any images I used that were not in Fog's files were listed as royalty free.  This and the "Generic The Fog" should round out all the buttons in the set, both active and awaiting their time.

![AgentOrange](https://user-images.githubusercontent.com/44233993/105805839-78089d00-5f68-11eb-95c2-a8c183382794.png)
![GButton](https://user-images.githubusercontent.com/44233993/105805842-78a13380-5f68-11eb-8f0e-8a424f4b8742.png)
![Huitzilopochtil](https://user-images.githubusercontent.com/44233993/105805843-7939ca00-5f68-11eb-8b58-263e1f33ce22.png)
![PhantomZero](https://user-images.githubusercontent.com/44233993/105805845-7939ca00-5f68-11eb-9a11-cb5c1e706903.png)
![PinballWizard](https://user-images.githubusercontent.com/44233993/105805846-7939ca00-5f68-11eb-8589-50d00ca47aeb.png)
![Steve](https://user-images.githubusercontent.com/44233993/105805848-79d26080-5f68-11eb-84d8-8feea188293b.png)
![TheGrotch](https://user-images.githubusercontent.com/44233993/105805849-79d26080-5f68-11eb-817b-59e8ea35aba2.png)

**Comment by TheOrgg** [comment]
Of this group, only Phantom Zero needs to be pushed to the main site.  The rest await Rebound.

**Comment by TheOrgg** [comment]
@blackshadowshade , didn't see anything about this on the Pulls.  Everything okay with it?

**Comment by blackshadowshade** [comment]
I'd love some comments on these buttons from testers, as always.  :)

**Comment by TheOrgg** [comment]
Okay.  I'll put a post up.  I think these came out pretty well considering what I had to work with (what Fog had in his files and G's artwork).  I think the shine is the only one I've done I'm happy with.

**Comment by dwvanstone** [comment]
The artwork looks good. It looks like the artwork for Steve doesn't exactly match what's on the site (I assume that's the Mad Batches?) but that's an issue for another day.

Having Phantom Zero artwork would be a great bonus! 

Was the "Generic The Fog" in another issue? If so, point to it in this issue please. Thanks!

**Comment by TheOrgg** [comment]
https://github.com/buttonmen-dev/buttonmen/issues/2693

**Comment by TheOrgg** [comment]
Also, thread for BSS:  http://www.buttonweavers.com/ui/forum.html#!threadId=937

**Comment by blackshadowshade** [comment]
Thanks for the comments. I'll put this in a pull request when I have a spare moment or two.

**Comment by cgolubi1** [comment]
It looks like this may have fallen through the cracks; Shadowshade, can you put in a PR for these next week, so we can make sure to get them in the next release?

**Comment by blackshadowshade** [comment]
Okay, I've whipped together a quick pull request. Make sure it's right, I was in a hurry (and being eaten alive by mosquitoes)!

**Comment by TheOrgg** [comment]
Phantom Zero is still missing artwork.  Should this be closed?

**Comment by blackshadowshade** [comment]
@TheOrgg, when you post something like this, it's useful to go through a number of steps:

1. Did you check the current codebase?

I've checked the current codebase and I see the image: https://github.com/buttonmen-dev/buttonmen/blob/master/src/ui/images/button/phantomzero.png

2. Has there been a deployment of the changes to a testing site?

Links to test sites will generally be visible on the issue itself, like here: https://github.com/buttonmen-dev/buttonmen/pull/2720#issuecomment-826993764

3. Has there been a recent deployment to staging.buttonweavers.com? To www.buttonweavers.com?

In this case, the answer is still no. We're waiting on @cgolubi1 to okay the last change for CustomBM, which was a bug fix of a bug found by RandomAI, and then she will do a deployment to staging.

To check the status of deployments, you'll see issues like #2733, where @cgolubi1 keeps us up to date with where she's at.


So, the current status is that you should see the image at staging once there is a deployment there.
